src/index.js

```diff
@@ -12,11 +12,14 @@ export const renderReact = (name, component) => hypernova({
   },
 
   client() {
-    const { node, data } = load(name);
+    const payloads = load(name);
 
-    if (node) {
-      const element = React.createElement(component, data);
-      ReactDOM.render(element, node);
+    if (payloads) {
+      Array.prototype.map.call(payloads, (payload) => {
```

Comment:
Could you explain it in detail? 
Cause I thought `load` just return something like `Array.prototype.map.call(nodes, () => ...)`
Should not the syntax above return an array? 
Or there is something else I missed? :P